playbooks/provisioning/openstack/post-provision-openstack.yml

```diff
@@ -90,3 +99,18 @@
         line: "IP4_NAMESERVERS={{ hostvars['localhost'].private_dns_server }}"
   roles:
     - node-network-manager
+
+- name: Show configured DNS nsupdate keys for future use as an external one
```

Comment:
I like the idea of showing the keys, but unfortunately I think this message may just be "lost" in the noise. My suggestion would be to just have another role that goes with the `dns-server` role (in `infra-ansible` ) that allows us to obtain/gather the keys and display them.